src/views/_init.js.njk

```diff
@@ -3,6 +3,16 @@ window.addEventListener('load', (event) => {
 
   if (window.GOVUKFrontend) {
     window.GOVUKFrontend.initAll();
+  if(document.querySelector('#current-species-caught')) {
+    accessibleAutocomplete.enhanceSelectElement({
+      selectElement: document.querySelector('#current-species-caught'),
+      confirmOnBlur: false,
+      classes: "{% if model.currentSpeciesCaughtError %} govuk-form-group--error {% endif %}",
+      defaultValue: "",
+      showAllValues: true
+    })
+  }
+
```

Comment:
I'm going to be really petty here, but the whitespace is off. The linter probably missed it since it's a `.js.njk` and not just a straight `.js` file.